shared/product-make.include

```diff
@@ -91,13 +91,13 @@ $(OUT)/xccdf-unlinked-ocilrefs.xml: $(OUT)/xccdf-unlinked-resolved.xml $(OUT)/oc
 	xsltproc -stringparam product "$(PROD)" -o $@ $(SHARED)/$(TRANS)/xccdf-ocilcheck2ref.xslt $<
 
 # Common build targets - a catalogue of XCCDF remediations
-bash_remediations_deps= $(wildcard $(IN)/remediations/bash/*.sh) $(wildcard $(SHARED_REMEDIATIONS)/*.sh)
+bash_remediations_deps= $(wildcard $(IN)/templates/static/bash/*.sh) $(wildcard $(SHARED_REMEDIATIONS)/*.sh)
 $(OUT)/bash-remediations.xml: $(SHARED)/$(TRANS)/combineremediations.py $(bash_remediations_deps)
 	# Make intermediate $(BUILD_REMEDIATIONS) directory to hold final list of remediation scripts for $(PROD)
 	mkdir -p $(BUILD_REMEDIATIONS)
-	# Search $(SHARED_REMEDIATIONS) and $(IN)/remediations directories to find all product specific remediation scripts,
+	# Search $(SHARED_REMEDIATIONS) and templates/static/bash directories to find all product specific remediation scripts,
 	# which are regular files (not symlinks). Merge the final list into $(BUILD_REMEDIATIONS) directory
-	find $(SHARED_REMEDIATIONS) $(IN)/remediations/bash -maxdepth 1 -type f -name '*.sh' -exec cp {} $(BUILD_REMEDIATIONS) ';'
+	find $(SHARED_REMEDIATIONS) templates/static/bash -maxdepth 1 -type f -name '*.sh' -exec cp {} $(BUILD_REMEDIATIONS) ';'
```

Comment:
Because it was in input directory. I should be `product/templates/...` because shared dir uses format `shared/templates`. Do you think that it should be `product/input/templates`?